fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java

```diff
@@ -432,6 +433,8 @@ public class Env {
 
     private PolicyMgr policyMgr;
 
+    private MtmvJobManager mtmvJobManager;
```

Comment:
`MTMV` is short for *Multiple Table Materialized Table*. I think to use `MTMV` (all capital characters) is better.